src/tests/ftest/util/ior_test_base.py

```diff
@@ -202,7 +204,15 @@ def verify_pool_size(self, original_pool_info, processes):
             processes (int): number of processes
         """
         # Get the current pool size for comparison
-        current_pool_info = self.pool.pool.pool_query()
+        if isinstance(self.pool, TestPool):
+            self.pool.get_info()
+            current_pool_info = self.pool.info
+        elif isinstance(self.pool, DaosPool):
```

Comment:
Since defining `self.pool` as a `DaosPool` object will cause other problems in this class I don't think it makes sense to support the alternate definition here.  If you agree, then the addition on line 33 can also be removed.

Reply:
As of nvme_io is using DaosPool which is also using this method. So until we move nvme_io to use TestPool as well I think we should keep this. In the changes for nvme_io when it is moved from using DaosPool to TestPool, we can remove the support for DaosPool from this method as well.